styles.css

```diff
@@ -288,3 +285,84 @@ button:hover,
 }
 
 /* About myself section */
+
+/* contact section */
+.contact-section {
+  background-color: #fff;
+  margin-left: -2.4rem;
+  margin-right: -3.2rem;
+}
+
+.contact-wrapper {
+  background-image: url(images/contact-form-shapes-mobile.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position-x: 100%;
+  background-color: #6070ff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-top-left-radius: 5.6rem;
+  padding: 5rem 1.5rem;
+}
+
+.contact-section h1 {
+  margin-top: 20%;
+  color: #fff;
+  font-size: 2.5rem;
+  line-height: 3.25rem;
+  font-weight: 700;
+  margin-bottom: 1rem;
+  text-align: center;
+}
+
+.contact-section p {
+  color: #ebebff;
+  font-size: 1.25rem;
+  font-weight: 400;
+  line-height: 1.75rem;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 0;
+  width: 100%;
+}
+
+form input[type="text"],
+input[type="email"],
+input[type="submit"],
+form textarea {
+  border-radius: 0.5rem;
+  padding: 1rem;
+  margin: 0.7rem 0;
+  border: 0;
+}
+
+form input[type="submit"] {
+  font-size: 1.07rem;
+  font-weight: 500;
+  line-height: 1.5rem;
+  color: #6070ff;
+  text-align: center;
+  padding: 0.75rem 1rem;
+  width: 8.9rem;
+}
+
+input[type="submit"]:hover,
+input[type="submit"]:focus {
+  color: #fff;
+  background-color: #6070ff;
+  transition: 0.5ms ease-in-out;
+  border: 1px solid #ffff;
```

Comment:
- [ ] According to to the template there is no border added to the button element. Kindly remove this border and your nice app would be approved. Reproducing perfectly according to the given design shows how well we pay attention to details.

Reply:
## Kindly check the button hover color and background color for contact form
I made this white color border to show the effect of  hover on submit button since the background color for contact is the same as the mouse hover 6070ff. 
So I'm adding the border effect to make my design interactive